src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,182 @@
+        fontSize: "12px",
+    },
+    card: {
+        margin: "0px 40px 20px",
+    },
+    table: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderRightWidth: 0,
+        borderBottomWidth: 0,
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
+        textTransform: "uppercase",
+        fontSize: "30px",
+        textAlign: "center",
+        margin: "30px",
+    },
+    dateLocationWrap: {
+        justifyContent: "space-between",
+        marginBottom: "5px",
+    },
+    svg: {
+        margin: "0 1px",
+        padding: "0 1px",
+    },
+    cell: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderLeftWidth: 0,
+        borderTopWidth: 0,
+        padding: "3px",
+    },
+    row: { display: "flex", flexDirection: "row" },
+    bold: { fontFamily: "Helvetica-Bold" },
+
+    // cells
+    cell_logo: { flex: 1 },
+    cell_name: { flex: 4 },
+    cell_postcode: { flex: 4 },
+    cell_time: { flex: 2 },
+    cell_collection: { flex: 4 },
+    cell_instructions: { flex: 10 },
+});
+
+const dateTimeToAMPM = (datetime: string): string => {
+    return new Date(datetime).getHours() <= 11 ? "AM" : "PM";
+};
+
+const CustomSVG: React.FC<CustomSVGProps> = ({ icon, color, fill }) => (
+    <Svg width={12} height={12} viewBox="0 0 512 512" style={styles.svg}>
+        <Path
+            fill={fill ? color : undefined}
+            stroke={color}
+            strokeWidth={40}
+            d={icon.icon[4] as string}
+        />
+        {/*IconDefinition[4] is defined as string | string[] but in reality there is no use case*/}
+        {/*with string[] as svgPathData*/}
+    </Svg>
+);
+
+const DayOverviewDate: React.FC<{ date: Date }> = ({ date }) => {
+    const dateString = date.toLocaleDateString("sv-SE");
+    return (
+        <View style={styles.row}>
+            <Text style={styles.bold}>Date: </Text> <Text>{dateString}</Text>
+        </View>
+    );
+};
+
+const DayOverviewLocation: React.FC<{ location: string }> = ({ location }) => (
+    <View style={styles.row}>
+        <Text style={styles.bold}>Location: </Text> <Text>{location}</Text>
+    </View>
+);
+
+const DayOverviewHeader: React.FC<{}> = () => {
+    return (
+        <View style={styles.row}>
+            <View style={[styles.cell_logo, styles.bold, styles.cell]}></View>
+            <View style={[styles.cell_name, styles.bold, styles.cell]}>
+                <Text style={styles.bold}>Name</Text>
+            </View>
+            <View style={[styles.cell_postcode, styles.bold, styles.cell]}>
+                <Text style={styles.bold}>Postcode</Text>
+            </View>
+            <View style={[styles.cell_time, styles.bold, styles.cell]}>
+                <Text style={styles.bold}>Time</Text>
+            </View>
+            <View style={[styles.cell_instructions, styles.bold, styles.cell]}>
+                <Text style={styles.bold}>Instructions</Text>
+            </View>
+        </View>
```

Comment:
Similarly to the one above, you could condense this down by not wrapping one text in one view. (I did not try this out on your code particularly but I condensed my code using this so I think it works in this case as well).

Reply:
Done